install.ps1

```diff
@@ -9,6 +9,40 @@ function Test-GitInstalled {
     }
 }
 
+function Compare-SemVer {
+    param (
+        [string]$version1,
+        [string]$version2
+    )
+
+    $version1Components = $version1 -split '\.'
+    $version2Components = $version2 -split '\.'
+
+    for ($i = 0; $i -lt 3; $i++) {
```

Comment:
Does this work with suffixes like 1.0.0-dev.1?

Reply:
Trying this on my windows machine now and it turns out git does not adhere to semver 🙃 The previous implementation worked, so I'm just going to rename the function to reflect what it actually does.

Reply:
(the version installed on my windows laptop is `2.41.0.windows.1`)